src/search.js

```diff
@@ -1,23 +1,25 @@
-import {assoc, concat, filter, has, isEmpty, map, pipe, prop, sortBy} from "ramda"
+import {ascend, assoc, concat, descend, has, isEmpty, map, prop, sortBy, sortWith} from "ramda"
 import * as diacritics from 'diacritics'
 
 
 export function findParametersAndVariables(parameters, variables, query) {
-  function matchesQuery(parameterOrVariable) {
-    if (isEmpty(normalizedQuery)) {
-      return true
-    }
-    return parameterOrVariable.name.includes(normalizedQuery) ||
-      parameterOrVariable.searchIndex.includes(normalizedQuery)
+  function indexInString(substring, string) {
+    const index = string.indexOf(substring)
+    return index === -1 ? Number.MAX_SAFE_INTEGER : index
   }
   const normalizedQuery = diacritics.remove(query.toLowerCase())
-  return pipe(
-    filter(matchesQuery),
-    sortBy(prop('name')),
-  )(concat(
+  let parametersAndVariables = concat(
     map(assoc('type', 'parameter'), parameters),
     map(assoc('type', 'variable'), variables),
```

Comment:
Indeed, that should have been deleted in #46.

It was used previously to determine the URLs which included `/variables` or `/parameters` prefix (ie `http://localhost:2030/variables/irpp`), but since we decided to have `http://localhost:2030/irpp` this is no more necessary to add a `type`.

But you're right, even previously, this should have been done in the caller.